samples/nodejs-compat-module/foo.js

```diff
@@ -0,0 +1,10 @@
+process.nextTick(() => {
```

Comment:
Note that this uses a different global scope than other modules in the bundle, just like `commonJsModule` types.

Reply:
Actually, this is wong! I forgot that these modules use a v8 mechanism called a context extension, which just extends the regular global scope... so we're good here :-)